[2025-03-12T23:31:40.219+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T23:31:40.295+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: processar_emails_excel_operador.concatenar_dataframes scheduled__2025-03-11T00:00:00+00:00 [queued]>
[2025-03-12T23:31:40.326+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: processar_emails_excel_operador.concatenar_dataframes scheduled__2025-03-11T00:00:00+00:00 [queued]>
[2025-03-12T23:31:40.342+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-12T23:31:40.713+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): concatenar_dataframes> on 2025-03-11 00:00:00+00:00
[2025-03-12T23:31:40.734+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=87) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-12T23:31:40.737+0000] {standard_task_runner.py:72} INFO - Started process 91 to run task
[2025-03-12T23:31:40.737+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'processar_emails_excel_operador', 'concatenar_dataframes', 'scheduled__2025-03-11T00:00:00+00:00', '--job-id', '162', '--raw', '--subdir', 'DAGS_FOLDER/extraiEmail.py', '--cfg-path', '/tmp/tmpnv4m_61w']
[2025-03-12T23:31:40.739+0000] {standard_task_runner.py:105} INFO - Job 162: Subtask concatenar_dataframes
[2025-03-12T23:31:40.821+0000] {task_command.py:467} INFO - Running <TaskInstance: processar_emails_excel_operador.concatenar_dataframes scheduled__2025-03-11T00:00:00+00:00 [running]> on host 4052a614e333
[2025-03-12T23:31:40.973+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='processar_emails_excel_operador' AIRFLOW_CTX_TASK_ID='concatenar_dataframes' AIRFLOW_CTX_EXECUTION_DATE='2025-03-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-11T00:00:00+00:00'
[2025-03-12T23:31:40.975+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-12T23:31:40.976+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-12T23:31:40.977+0000] {logging_mixin.py:190} INFO - Current task name:concatenar_dataframes state:running start_date:2025-03-12 23:31:40.300276+00:00
[2025-03-12T23:31:40.978+0000] {logging_mixin.py:190} INFO - Dag name:processar_emails_excel_operador and current dag run status:running
[2025-03-12T23:31:40.978+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-12T23:31:40.995+0000] {logging_mixin.py:190} INFO - Nenhum DataFrame encontrado para converter.
[2025-03-12T23:31:40.996+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-12T23:31:41.013+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-12T23:31:41.014+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=processar_emails_excel_operador, task_id=concatenar_dataframes, run_id=scheduled__2025-03-11T00:00:00+00:00, execution_date=20250311T000000, start_date=20250312T233140, end_date=20250312T233141
[2025-03-12T23:31:41.028+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-12T23:31:41.029+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-12T23:31:41.030+0000] {logging_mixin.py:190} INFO - Dag name:processar_emails_excel_operador queued_at:2025-03-12 23:31:01.218508+00:00
[2025-03-12T23:31:41.031+0000] {logging_mixin.py:190} INFO - Task hostname:4052a614e333 operator:PythonOperator
[2025-03-12T23:31:41.077+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T23:31:41.102+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-12T23:31:41.106+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
